build/release/README.md

```diff
@@ -71,3 +101,23 @@ Ensure that you only click `Save draft` until the release is complete, after whi
 ## Release Artifacts
 
 Images are pushed to docker hub under the [rook/ceph](https://hub.docker.com/r/rook/ceph/tags/) repo.
+
```

Comment:
Did you intend to remove this section? I assume it was temporary.

Reply:
Woops, yep, removed it now.